src/components/Boards/Boards.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Box from '@material-ui/core/Box';
+import * as S from './Boards.styles';
```

Comment:
Do you think using the full name (like `Styles`) instead of `S` can help with readability?

Reply:
I feel like S is short and sweet.